lib/ansible/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -213,7 +215,7 @@ def main():
             type=dict(required=False, choices=['metric alert', 'service check', 'event alert', 'process alert']),
             name=dict(required=True),
             query=dict(required=False),
-            message=dict(required=False, default=None),
+            notification_message=dict(required=False, default=None, aliases=['message']),
```

Comment:
Why not use deprecated_aliases, 

```suggestion
            notification_message=dict(required=False, default=None, 
                deprecated_aliases=[dict(name='message', version='2.14')])
```